### PR TITLE
Bump docker to 9.3.4

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM postgres:12.2
```
Comment (hanefi) [inline Dockerfile:1]: `postgres:12.3` is out. Shall we upgrade?

Comment (onurctirtir) [reply Dockerfile:1]: I guess we bump postgres version in major release, is this not correct ?

**Comment by onurctirtir** [comment]
Should we merge this ?

File: Dockerfile (hunk 1 above)
Comment (hanefi) [reply Dockerfile:1]: Not neccessarily. We usually try to bump PG version once it becomes available.

**Pull request merged**